src/coordinate_conversions.cpp

```diff
@@ -177,8 +179,37 @@ point ms_to_omt_remain( int &x, int &y )
     return point( divide( x, SEEX * 2, x ), divide( y, SEEY * 2, y ) );
 }
 
-
 tripoint omt_to_seg_copy( const tripoint &p )
 {
-    return tripoint( divide( p.x, 32 ), divide( p.y, 32 ), p.z );
+    return tripoint( divide( p.x, SEG_SIZE ), divide( p.y, SEG_SIZE ), p.z );
+}
+
+void limit_coordinates( int &x, int &y )
+{
+    if( get_option<int>( "WORLD_LIMIT_X" ) > 0 ) {
+        x = x % get_option<int>( "WORLD_LIMIT_X" );
+    }
+    if( get_option<int>( "WORLD_LIMIT_Y" ) > 0 ) {
+        y = y % get_option<int>( "WORLD_LIMIT_Y" );
+    }
+}
+
+void limit_coordinates( point &p )
```

Comment:
Please don't repeat the whole logic in those two functions, just forward `p.x` and `p.y` to the main function above.

Also, the name of those functions is not good, what coordinates are limited and how and why? Previously (the previous version of this branch) the name made some sense as it was scoped to `overmap_buffer`, now it's a global function and one does not know what coordinates it refers to. It should at least contain "om" or "overmap" to indicate it operates on overmap coordinates.

The function declaration in the header is also missing some comment explaining why and when to use it and what coordinates to use it on. (Again, this is not so important when it's a local, or even private, function of a class, but now it's global and everywhere may see and use it.)

Reply:
Thanks. I've utilized inline functions to get rid of duplicate code.

The reason I moved these functions is that it seems to me to be correct place as new functions are also related to coordinate conversions. I didn't come up with the good name yet - will try something mentioning `overmap`.

Anyway that should've been changes to temporary local branch, but I accidentally committed to wrong branch and also pushed the changes.